### PR TITLE
[meshmap] add boilerplate code for cxtmenu working with react-cytoscape

### LINKED ISSUE #1462: [MeshMap] Context menu for performance tests
**Current Behavior**

**Desired Behavior**

---
**Implementation**

**Mockups**

**Acceptance Tests**

### PR DESCRIPTION
**Pull request by anirudhjain75**
**Description**

This PR fixes #1462 

**Notes for Reviewers**
I've used template code for command names and they are subject to change based on requirement

### CODE REVIEW

File: ui/components/MesheryVisualizeComponent.js [hunk 1]
```diff
@@ -11,9 +15,60 @@ class MesheryVisualizeComponent extends React.Component {
       { data: { source: 'one', target: 'two', label: 'Edge from Node1 to Node2' } }
     ];
 
+    let cxtMenuSettings = {
+      menuRadius: 100, // the radius of the circular menu in pixels
+      selector: 'node', // elements matching this Cytoscape.js selector will trigger cxtmenus
+      commands: [ // an array of commands to list in the menu or a function that returns the array
+        {
+          fillColor: 'rgba(200, 200, 200, 0.75)', // optional: custom background color for item
```
Comment (leecalcote) [inline ui/components/MesheryVisualizeComponent.js:23]: Depending upon how we bring Meshery and MeshMap together, this can reference a global variable.

**Comment by leecalcote** [comment]
@anirudhjain75 there are a couple conflicts that need to be addressed.

**Comment by dhruv0000** [comment]
@anirudhjain75 We had created a separate folder in the components folder for visualise. May be the reason for the conflict.

**Comment by anirudhjain75** [comment]
Merge conflict is for merging with master.... Not with meshmap branch.

**Comment by leecalcote** [comment]
> Merge conflict is for merging with master.... Not with meshmap branch.

Irrespective, GitHub isn't going to allow merging until they are resolved.

**Comment by kushthedude** [comment]
> Merge conflict is for merging with master.... Not with meshmap branch.

Fetch the upstream meshmap, I think @dhruv0000 has put visualise compnenents in a sub-folder for better sanitisation I guess.

**Comment by anirudhjain75** [comment]
@leecalcote can you update mesmaP with latest master. I don't have wrote access to repo

**Comment by kushthedude** [comment]
> @leecalcote can you update mesmaP with latest master. I don't have wrote access to repo

Done, update your branch now!

**Comment by leecalcote** [comment]
Thank you, gents.

**Pull request merged**